src/markdown-pages/install-with-kurl/removing-minio.md

```diff
@@ -0,0 +1,71 @@
+---
+path: "/docs/install-with-kurl/removing-minio"
+date: "2021-12-17"
+weight: 21
+linktitle: "Removing Minio"
+title: "Removing Minio from your Cluster"
+---
+
+As of [v2021.12.20-0](https://kurl.sh/release-notes/v2021.12.20-0), kURL clusters can be installed without add-ons having a peer dependency on object storage. 
+There is also a migration path for clusters to remove object storage, such as is the case with removing Minio.
+
+# New Installations
+
+The following add-ons can be installed without object storage:
```

Comment:
Lots of add-ons can be installed without object storage. I like this wording since it shows that they can use object storage but now don't require it.
```suggestion
The following add-ons can now be installed with or without object storage:
```

Reply:
Yeah, I definitely think I missed a word here. :thumbsup: